### DISCUSSION TITLE
Conditional Card should allow for "Greater than" and "Less Than" in its conditions

### DISCUSSION THREAD

**Discussion by realtek2017** [home-assistant/frontend#10044] [Dashboard]
Currently the only filters available are State is "Equal To" or "State is Not Equal To"

Conditional Card should allow for "Greater than" and "Less Than" in its conditions.

This is particularity useful for values such as battery values to perhaps show a entity when the Battery level drops below a certain value

**Comment by spacegaier** [comment]
There were attempts but they somehow stalled out:

https://github.com/home-assistant/frontend/pull/7381
https://github.com/home-assistant/frontend/pull/7688

Should probably be revisited.
